From 68218ae871ef93b8b86f4123e333f89614fe87e4 Mon Sep 17 00:00:00 2001
From: Vinitha Pillai-B57223 <vinitha.pillai@nxp.com>
Date: Thu, 23 Nov 2017 17:11:17 +0530
Subject: [PATCH 204/211] LS1012ARDB Secure Boot: Add encapsulation support

"setexpr" command is needed as a part of encapsulation
feature, to calculate size of new encapsulated image.
CONFIG_CMD_SETEXPR is set by default, which is disabled by using
"# CONFIG_CMD_SETEXPR is not set" in defconfig, so removed it
to enable the command.

Signed-off-by: Vinitha Pillai-B57223 <vinitha.pillai@nxp.com>
---
 configs/ls1012ardb_qspi_SECURE_BOOT_defconfig |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/configs/ls1012ardb_qspi_SECURE_BOOT_defconfig b/configs/ls1012ardb_qspi_SECURE_BOOT_defconfig
index 09fc624..af9fb16 100644
--- a/configs/ls1012ardb_qspi_SECURE_BOOT_defconfig
+++ b/configs/ls1012ardb_qspi_SECURE_BOOT_defconfig
@@ -18,7 +18,6 @@ CONFIG_CMD_MMC=y
 CONFIG_CMD_SF=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_USB=y
-# CONFIG_CMD_SETEXPR is not set
 CONFIG_CMD_DHCP=y
 CONFIG_CMD_MII=y
 CONFIG_CMD_PING=y
-- 
1.7.1

